Vates/VatesAPI/inc/MantidVatesAPI/PresenterFactories.h

```diff
@@ -13,15 +13,15 @@ class WorkspaceProvider;
 
 class DLLExport EmptyWorkspaceNamePolicy {
 protected:
-  std::string getWorkspaceName(Mantid::API::IMDWorkspace_sptr) {
-    return "__EmptyWorkspaceNamePolicy";
-  }
+  const std::string &
+  getWorkspaceName(const Mantid::API::IMDWorkspace & /*workspace*/);
```

Comment:
@quantumsteve Does Doxygen cope with the commented out names?

My preference would be `UNUSED_ARG` as it's in more active use (I think) but I don't have strong feelings either way.